include/ibamr/IBFESurfaceMethod.h

```diff
@@ -89,17 +89,33 @@ namespace IBAMR
  * \brief Class IBFESurfaceMethod is an implementation of the abstract base
  * class IBStrategy that provides functionality required by the IB method with
  * a finite element representation of a surface mesh.
+ *
+ * Coupling schemes include both IB formulations (integral operations with
+ * regularized delta function kernels) and an immersed interface method (IIM)
+ * scheme (E. M. Kolahdouz, A. P. S. Bhalla, B. A. Craven, and B. E. Griffith.
+ * An immersed interface method for discrete surfaces. J Comput Phys,
+ * 400:108854 (37 pages), 2020).
+ *
+ * \note When using the IIM implementation, it is recommended that users set
+ * all linear solvers to use tight relative tolerances (1e-10).
  */
 class IBFESurfaceMethod : public IBStrategy
 {
 public:
-    static const std::string COORDS_SYSTEM_NAME;
     static const std::string COORD_MAPPING_SYSTEM_NAME;
+    static const std::string COORDS_SYSTEM_NAME;
     static const std::string FORCE_SYSTEM_NAME;
     static const std::string NORMAL_VELOCITY_SYSTEM_NAME;
+    static const std::string PRESSURE_IN_SYSTEM_NAME;
```

Comment:
One way to reduce the number of systems would be to have separate variables for the "inner" and "outer" reconstructions in the same `System` (e.g. variable `0` is inner and `1` is outer).

Reply:
That's a good idea. I think at the end of this set of patches we should try to minimize the number of systems but it's not a big deal - I think it would be better to keep extra systems for now to avoid extra breakage.